docs/use-cases/control-motor.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Control a motor in 60 seconds"
+linkTitle: "Control a motor in 60 seconds"
+type: "docs"
+weight: 30
+images: ["/icons/components/motor.svg"]
+description: "Use Viam to control a motor's speed and direction in just a few steps."
+---
+
+This guide will demonstrate how to control a motor using Viam in just a few steps. You can use Viam to control a motor's speed and direction directly from [the platform](https://app.viam.com/), [programatically](https://docs.viam.com/build/program/), or by using [the mobile app](/fleet/#the-viam-mobile-app).
+
+To control your motor using Viam, refer to the following instructions:
+
+{{< expand-and-search "Step 1: Configure a Board" >}}
+
+First, [create a machine](/cloud/machines/#add-a-new-machine) if you haven't yet.
+
+Then, [add a board component](/components/board/), such as a [Raspberry Pi board](/components/board/pi/).
```

Comment:
hmmm do you think you could actually get this done in 60 seconds. We may need to adjust this. We may also need to add a requirements section that says you need to have a board with a supported OS installed and a motor connected.